src/index.ts

```diff
@@ -24,10 +26,10 @@ const initialised = new Promise((resolve) => {
 
 function init({
 	pubData,
-	isInUsa,
+	framework,
```

Comment:
this feels like it asks consumers of CMP to know a lot about decisions the business has made, and how the CMP works internally i.e. what frameworks are there, and which one applies where?

i _think_ we only need to know where the user is? 

so rather than `isInUsa` we could require something like `location: 'US' | 'AU' | 'RoW'`?

Reply:
Yes, I agree!

I was also thinking we should accept any country code, rather than `RoW`. This would mean, for example, that a `NZ` or `CA` user could be put in a different and specific framework without any of the code consuming `@guardian/consent-management-platform` to change at all.

Reply:
oh yeah great idea